app/assets/javascripts/arclight/collection_navigation.js

```diff
@@ -30,12 +30,20 @@
         var $doc = $(resp);
         var showDocs = $doc.find('article.document');
         var newDocs = $doc.find('#documents');
-
         var sortPerPage = $doc.find('#sortAndPerPage');
+        var pageEntries = sortPerPage.find('.page-entries');
+        var numberEntries = parseInt(pageEntries.find('strong').last().text().replace(/,/g, ''), 10);
+
         // Hide these until we re-enable in the future
         sortPerPage.find('.result-type-group').hide();
         sortPerPage.find('.search-widgets').hide();
 
+        if (!isNaN(numberEntries)) {
+          $('[data-arclight-online-content-tab]').append(
+            $('<span class="badge badge-pill badge-secondary al-online-content-badge">' + numberEntries + '</span>')
```

Comment:
Do we need to add some `sr-only` text after this number? We add "components" after the badge number for content. Maybe "items", @jvine?

![badge_number](https://user-images.githubusercontent.com/5402927/67331608-d4158380-f4d2-11e9-9f36-6ec2e836f231.png)


```html
<span class="badge badge-pill badge-secondary al-number-of-children-badge">
	25<span class="sr-only">components</span>
</span>
```